src/ComposerRequireChecker/Cli/CheckCommand.php

```diff
@@ -53,13 +53,15 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        if (!$output->isQuiet()) {
+        if (!$output->isQuiet() && $this->getApplication() !== null) {
```

Comment:
Let's avoid repetition of calls: PHP doesn't guarantee them to be idempotent. Instead, let's cache the application into a variable in local scope